diff --git a/node_modules/isomorphic-ws/index.d.ts b/node_modules/isomorphic-ws/index.d.ts
index de923a9..3b57081 100644
--- a/node_modules/isomorphic-ws/index.d.ts
+++ b/node_modules/isomorphic-ws/index.d.ts
@@ -3,6 +3,6 @@
 // Fix for https://github.com/heineiuo/isomorphic-ws/issues/8
 // If there is still something wrong, welcome issue.
 
-import WebSocket = require('ws')
+import { WebSocket } from 'ws';
 
-export = WebSocket
\ No newline at end of file
+export default WebSocket;
\ No newline at end of file
